docs/running_psalm/error_levels.md

```diff
@@ -4,6 +4,9 @@ You can run Psalm in at different levels of strictness from 1 to 8.
 
 Level 1 is the most strict, level 8 is the most lenient.
 
+When no level is explicitly defined, psalm defaults to level 2.
+In case `totallyTyped` is enabled, psalm defaults to level 1.
```

Comment:
Since totallyTyped is deprecated we might just drop this line for simplicity sake